Create ru_ru.json

shikhtv opened the pull request:
Translated the names of all blocks and items. Added descriptions to machines. It remains to translate 20% of the text.

shikhtv committed:
Create ru_ru.json

Translated the names of all blocks and items. Added descriptions to machines. It remains to translate 20% of the text.

desht commented:
Thanks for this.  Would you prefer to get the remaining translations in before I pull it, or should I just pull now and pull any updates later?

shikhtv commented:
You can now. And the update will be a little later

shikhtv committed:
Update ru_ru.json

Updated translation by 100%.

shikhtv committed:
Update ru_ru.json

desht commented:
Thanks, I'll get this in for the next release.

shikhtv committed:
Update ru_ru.json

some more fixes. in some incomprehensible way, my software has changed the places of some lines ... I think now everything.
